lms/templates/instructor/instructor_dashboard_2/certificates.html

```diff
@@ -1,6 +1,9 @@
 <%page args="section_data" expression_filter="h"/>
 <%namespace name='static' file='../../static_content.html'/>
 
+<%def name="online_help_token()"><% return "instructorcerts" %></%def>
```

Comment:
Unfortunately this method of setting individual pages based on which "tab" of the Instructor Dashboard is showing does not work. The instructor dashboard renders with instructor_dashboard_2.html, which then includes the individual "tab" HTML. The online_help_token is used at the time that the page is first rendered to construct the link for the Help button. Subsequent changes between different tabs will not update the Help URL at all.

I talked to @catong, and for the purposes of incremental improvement I'm going to strip out the attempts to make different tabs go to different locations.